Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM golang:latest
 
-COPY . /go
+WORKDIR /app
 
-RUN chmod +x /go/entrypoint.sh
+COPY go.mod ./
+RUN go mod download
 
-ENTRYPOINT [ "/go/entrypoint.sh" ]
\ No newline at end of file
+COPY . ./
+RUN chmod +x /app/entrypoint.sh
```

Comment:
It looks like this line is necessary because entrypoint.sh doesn't actually have its execute bit set in the repository.